src/Sarif/Writers/SarifLogger.cs

```diff
@@ -447,12 +447,16 @@ private void CaptureArtifact(ArtifactLocation fileLocation)
                 catch (ArgumentException) { } // Unrecognized encoding name
             }
 
+            HashData hashData = null;
+            AnalysisTargetToHashDataMap?.TryGetValue(fileLocation.Uri.LocalPath, out hashData);
```

Comment:
>LocalPath [](http://example.com/codeflow?start=70&length=9)

Is this data initialized for every kind of URI? have you tried testing relative URIs? Isn't it true that local path is only available to absolute URIs?